recipes/runit.rb

```diff
@@ -13,20 +13,24 @@
 node.set['runit']['sv_dir']       = "#{install_path}/sv"
 node.set['runit']['lsb_init_dir'] = "#{install_path}/init"
 
-case node['platform_family']
-when 'debian'
-  include_recipe 'enterprise::runit_upstart'
-when 'fedora', 'rhel'
-  case node['platform']
-  when 'amazon', 'fedora'
+if node['init_package'] == 'systemd'
+  include_recipe 'enterprise::runit_systemd'
```

Comment:
node['init_package'] is set by ohai and will == 'systemd' when pid 1 is systemd.  this seems correct to me.